ui/logviewer.html

```diff
@@ -12,40 +12,97 @@
+            <!-- Job status -->
+            <li class="{{::resultStatusShading}}">
+              <div>
+                <span ng-cloak><strong>{{result.label}}: </strong></span>
+                <span ng-cloak class="break-word">{{result.value}}</span>
+              </div>
+            </li>
+
+            <!-- Raw log button -->
+            <li class="logviewer-actionbtn">
+              <a title="Open the raw log in a new window"
+                 target="_blank"
+                 href="{{::artifact.logurl}}">
+                <span class="fa fa-file-text-o actionbtn-icon"></span>
+                <span>open raw log</span>
+              </a>
+            </li>
+
+            <!-- Ref test button -->
+            <li ng-if="isReftest()"
+                class="logviewer-actionbtn">
+              <a title="Open the Reftest Analyser in a new window"
+                 target="_blank"
+                 href="http://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl={{::artifact.logurl}}&only_show_unexpected=1">
+                <span class="fa fa-bar-chart-o actionbtn-icon"></span>
+                <span>open analyser</span>
+              </a>
+            </li>
+
+            <!-- Show successful steps button -->
+            <li ng-if="artifact && hasFailedSteps()"
+                class="logviewer-actionbtn">
+              <div id="lv-successful-steps">
+                <input type="checkbox"
+                       ng-model="showSuccessful"
+                       ng-change="toggleSuccessfulSteps()" />
+                <span>show successful steps</span>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <!-- Job information and Steps buttons -->
+      <div class="run-data">
+        <div class="col-md-6" >
+          <div class="job-header">
+            <table class="table table-condensed" >
+              <tr ng-repeat="property in logProperties">
+                <th ng-cloak>{{property.label}}</th>
+                <td ng-if="property.label == 'Revision'" class="break-word">
+                  <a href="{{::logRevisionFilterUrl}}"
+                     title="Open resultset"
+                     class="repo-link"
+                     ng-cloak>{{property.value}}</a>
+                </td>
+                <td ng-if="property.label != 'Revision'"
+                    ng-cloak class="break-word">{{property.value}}</td>
+              </tr>
+              <tr ng-repeat="line in job_details | orderBy:'title'">
+                <th>{{line.title}}:</th>
+                <td ng-switch on="line.content_type">
+                  <a ng-switch-when="link" title="{{line.value}}"
+                     href="{{line.url}}" target="_blank">{{line.value}}</a>
+                    <span ng-switch-when="raw_html" ng-bind-html="line.value"></span>
+                <td/>
+              </tr>
+            </table>
+          </div>
```

Comment:
So I went with our agreed 2 char html indentation, and I will convert the rest of logviewer.html to 2 space in a follow up PR immediately after this one.